tools/packaging/scripts/configure-hypervisor.sh

```diff
@@ -213,7 +213,7 @@ generate_qemu_options() {
 	if [ $arch != $host ];then
 		case $arch in
 			aarch64) qemu_options+=(size:--cross-prefix=aarch64-linux-gnu-);;
-			ppc64le) qemu_options+=(size:--cross-prefix=powerpc64le-linux-gnu-);;
+			ppc64le) qemu_options+=(size:--cross-prefix=powerpc64-linux-gnu-);;
```

Comment:
This is a prefix for the compiler that should remain `powerpc64le` according to :

https://packages.ubuntu.com/oracular/all/gcc-powerpc64le-linux-gnu/filelist

https://packages.fedoraproject.org/pkgs/cross-gcc/gcc-powerpc64le-linux-gnu/fedora-rawhide.html#files